peep.sh

```diff
@@ -8,27 +8,17 @@ ARGS=""
 
 echo "peep.sh: Using pip $PIPVER"
 
-# If we're using pip 1.5, 1.6 or 6.0, then toss on the --no-use-wheel
-# argument.
-#
-# Note: If we encounter other versions of pip here that require other
-# things, we can toss them in.
+# Custom arguments required for some versions of pip.
 case $PIPVER in
-    1.5*|1.6*|6.0*)
-        echo "peep.sh: Wheel-using pip detected, so passing --no-use-wheel."
+    1.5*)
+        # Wheel support is broken in pip < v6.0 due to:
+        # https://github.com/pypa/pip/issues/1825
+        echo "peep.sh: Pip with broken wheel support detected, so passing --no-use-wheel."
```

Comment:
This changes what this section was for. We don't want to use wheels, so this section passes the appropriate flag to versions of pip that support wheels. It's not about broken wheel support.